### PR TITLE
Tslint to Eslint Migration

### PR DESCRIPTION
**Pull request by vashian**

### CODE REVIEW

File: src/commands/app/shell.ts [hunk 1]
```diff
@@ -69,7 +69,7 @@ export default class AppShell extends Command {
       duplex.pipe(process.stdout);
 
       process.stdin.on('data', function (key) {
-        if (key.toString() == CTRL_Q) {
+        if (key.toString() === CTRL_Q) {
           clearStdinEffects();
```
Comment (vashian) [inline src/commands/app/shell.ts:73]: I'm not sure why you used two equal signs here.

File: src/utils/merge-platform-config.ts [hunk 2]
```diff
@@ -144,18 +145,20 @@ function getRequiredPHPVersion(
...
+
     throw error;
   }
 }
 
 // https://getcomposer.org/doc/articles/versions.md#version-range
 function convertSinglePipeToDouble(input: string) {
-  return input.replace(/\|{1,}/g, '||');
+  return input.replace(/\|+/g, '||');
 }
```
Comment (vashian) [inline src/utils/merge-platform-config.ts:156]: **Better Regex** ?

File: src/commands/app/shell.ts (hunk 1 above)
Comment (mhemrg) [reply src/commands/app/shell.ts:73]: I haven't. I guess Mohammad Amin added these lines.

**Comment by mhemrg** [comment]
Well done :100:

**Pull request merged**